.github/workflows/assign_label.yml

```
@@ -15,7 +15,12 @@ on:
       - synchronize
 
   workflow_call:
-
+    inputs:
+      runs-on:
+        type: string
+        required: false 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)